daisi/src/path_planning/agv_info.h

```diff
@@ -30,7 +31,7 @@ namespace daisi::path_planning {
 struct AGVInfo {
   std::string id;
   std::string ip;
-  cpps::Kinematics kinematics;
+  cpps::AmrKinematics kinematics;
```

Comment:
use `AmrDescription` or additional `AmrLoadHandlingUnit` to store the same information.